src/PluginManager/Microsoft.Performance.Toolkit.PluginManager.Core/Registry/InstalledPlugin.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Performance.Toolkit.PluginManager.Core.Discovery;
+
+namespace Microsoft.Performance.Toolkit.PluginManager.Core.Registry
+{
+    // TODO:
+    public sealed class InstalledPlugin : IPluginInfo
```

Comment:
`InstalledPlugin : IPluginInfo` doesn't make sense to me. I feel `IPluginInfo` should be a property on `InstalledPlugin`